### ISSUE TITLE
Issues using vagrant for local development: grunt/bower and phantomjs won't install

### ISSUE THREAD

**Issue by AnitaYCheng** [HabitRPG/habitica#5876]
Recloned the repo to clean out my Git setup, having trouble setting up my local environment with Vagrant with this latest clone. Spent the last few days repeating `vagrant destroy` `rm -rf node_modules` `vagrant up` with no avail.

On `vagrant up` command, the following errors occur (sorry, this is going to be LONG):

```
==> default: Setting up nodejs (0.10.37-1chl1~precise1) ...
==> default: Installing grunt/bower...
==> default: npm
==> default:  
==> default: ERR!
==> default:  Error: Attempt to unlock phantomjs, which hasn't been locked
==> default: npm
==> default:  
==> default: ERR!
==> default:      at unlock (/usr/lib/node_modules/npm/lib/utils/locker.js:44:11)
==> default: npm
==> default:  
==> default: ERR!
==> default:      at cb (/usr/lib/node_modules/npm/lib/cache/add-local.js:30:5)
==> default: npm
==> default:  
==> default: ERR!
==> default:      at /usr/lib/node_modules/npm/lib/cache/add-local.js:47:20
==> default: npm
==> default:  
==> default: ERR!
==> default:      at /usr/lib/node_modules/npm/lib/utils/locker.js:30:7
==> default: npm
==> default:  
==> default: ERR!
==> default:      at cb (/usr/lib/node_modules/npm/node_modules/lockfile/lockfile.js:149:38)
==> default: npm
==> default:  
==> default: ERR!
==> default:      at /usr/lib/node_modules/npm/node_modules/lockfile/lockfile.js:177:38
==> default: npm
==> default:  
==> default: ERR!
==> default:      at Object.oncomplete (fs.js:108:15)
==> default: npm
==> default:  
==> default: ERR!
==> default:  If you need help, you may report this *entire* log,
==> default: npm
==> default:  
==> default: ERR!
==> default:  including the npm and node versions, at:
==> default: npm
==> default:  
==> default: ERR!     <http://github.com/npm/npm/issues>
==> default: 
==> default: npm ERR! System Linux 3.2.0-23-generic
==> default: npm
==> default:  
==> default: ERR!
==> default:  
==> default: command
==> default:  "/usr/bin/node" "/usr/bin/npm" "install" "-g" "grunt-cli" "bower" "phantomjs"
==> default: npm
==> default:  ERR! cwd /vagrant
==> default: npm ERR! node -v v0.10.37
==> default: npm ERR! npm -v 1.4.28
==> default: npm
==> default:  
==> default: ERR!
==> default:  
==> default: npm
==> default:  
==> default: ERR!
==> default:  Additional logging details can be found in:
==> default: npm
==> default:  
==> default: ERR!
==> default:      /vagrant/npm-debug.log
==> default: npm
==> default:  
==> default: ERR!
==> default:  
==> default: not ok
==> default:  code 0
==> default: Installing Habitica
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: package.json
==> default:  icalendar@0.6.4 No repository field.
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: package.json
==> default:  karma-phantomjs-launcher@0.1.4 No README data
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: package.json
==> default:  expect.js@0.2.0 No repository field.
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: package.json
==> default:  pretty-data@0.40.0 Invalid filename in 'files' list: 
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: cannot run in wd
==> default:  habitrpg@0.0.0-152 ./node_modules/bower/bin/bower --config.interactive=false install -f; ./node_modules/.bin/grunt; (wd=/vagrant)
==> default: npm
==> default:  WARN unmet dependency /vagrant/node_modules/phantomjssmith requires async@'~0.2.7' but will load
==> default: npm WARN unmet dependency /vagrant/node_modules/async,
==> default: npm WARN unmet dependency
==> default:  which is version 0.9.2
==> default: npm
==> default:  
==> default: WARN
==> default:  unmet dependency /vagrant/node_modules/protractor requires request@'~2.36.0' but will load
==> default: npm WARN unmet dependency /vagrant/node_modules/request,
==> default: npm WARN unmet dependency which is version 2.44.0
==> default: npm 
==> default: WARN unmet dependency /vagrant/node_modules/protractor requires glob@'~3.2' but will load
==> default: npm WARN unmet dependency /vagrant/node_modules/glob,
==> default: npm WARN unmet dependency which is version 4.5.3
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/protractor requires q@'1.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/q,
==> default: npm WARN unmet dependency which is version 1.4.1
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/bower/node_modules/insight/node_modules/inquirer/node_modules/cli-color/node_modules/d requires es5-ext@'~0.10.2' but will load
==> default: npm
==> default:  
==> default: WARN unmet dependency undefined,
==> default: npm WARN
==> default:  unmet dependency which is version undefined
==> default: npm WARN unmet dependency /vagrant/node_modules/bower/node_modules/insight/node_modules/inquirer/node_modules/cli-color/node_modules/timers-ext requires es5-ext@'~0.10.2' but will load
==> default: npm 
==> default: WARN
==> default:  unmet dependency undefined,
==> default: npm
==> default:  
==> default: WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/bower/node_modules/update-notifier/node_modules/configstore/node_modules/js-yaml/node_modules/argparse requires lodash@'>= 3.2.0 < 4.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/lodash,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version 2.4.2
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/browserify/node_modules/browser-pack/node_modules/JSONStream requires through@'~2.2.7' but will load
==> default: npm
==> default:  WARN unmet dependency /vagrant/node_modules/browserify/node_modules/through,
==> default: npm WARN unmet dependency which is version 2.3.8
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/express/node_modules/connect/node_modules/multiparty/node_modules/stream-counter requires readable-stream@'~1.1.8' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/concat-stream requires inherits@'~2.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/concat-stream requires readable-stream@'~1.1.9' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/crypto-browserify requires inherits@'^2.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/duplexer2 requires readable-stream@'~1.1.9' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/glob requires inherits@'2' but will load
==> default: npm WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm WARN unmet dependency /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/http-browserify requires inherits@'~2.0.1' but will load
==> default: npm WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/labeled-stream-splicer requires inherits@'^2.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/labeled-stream-splicer requires isarray@'~0.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/module-deps requires inherits@'^2.0.1' but will load
==> default: npm
==> default:  WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/module-deps requires readable-stream@'^1.1.13' but will load
==> default: npm WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/read-only-stream requires readable-stream@'^1.0.31' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/stream-browserify requires inherits@'~2.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm WARN 
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/stream-browserify requires readable-stream@'^1.0.27-1' but will load
==> default: npm
==> default:  WARN 
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  WARN unmet dependency which is version undefined
==> default: npm WARN unmet dependency /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/through2 requires readable-stream@'>=1.1.13-1 <1.2.0-0' but will load
==> default: npm WARN unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/browserify/node_modules/util requires inherits@'2.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/watchify/node_modules/browserify requires glob@'^4.0.5' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/glob,
==> default: npm
==> default:  
==> default: WARN
==> default:  unmet dependency which is version 5.0.14
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-browserify/node_modules/watchify/node_modules/browserify requires through2@'^1.0.0' but will load
==> default: npm
==> default:  WARN unmet dependency /vagrant/node_modules/grunt-browserify/node_modules/watchify/node_modules/through2,
==> default: npm WARN unmet dependency which is version 0.6.5
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-spritesmith/node_modules/spritesmith/node_modules/pixelsmith/node_modules/concat-stream/node_modules/readable-stream requires inherits@'~2.0.1' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/in-app-purchase/node_modules/request/node_modules/form-data requires mime@'~1.2.11' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/pageres/node_modules/protocolify/node_modules/file-url/node_modules/meow/node_modules/indent-string requires minimist@'^1.1.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/pageres/node_modules/viewport-list/node_modules/meow/node_modules/indent-string requires minimist@'^1.1.0' but will load
==> default: npm WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-build/node_modules/decompress/node_modules/decompress-tar/node_modules/tar-stream/node_modules/bl requires readable-stream@'~2.0.0' but will load
==> default: npm
==> default:  WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-build/node_modules/decompress/node_modules/decompress-targz/node_modules/tar-stream/node_modules/bl requires readable-stream@'~2.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-build/node_modules/download/node_modules/caw requires object-assign@'^3.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  unmet dependency /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-build/node_modules/download/node_modules/object-assign,
==> default: npm WARN unmet dependency which is version 2.1.1
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-wrapper/node_modules/download/node_modules/gulp-decompress/node_modules/decompress/node_modules/decompress-unzip requires vinyl@'^0.5.0' but will load
==> default: npm
==> default:  
==> default: WARN unmet dependency /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-wrapper/node_modules/download/node_modules/vinyl,
==> default: npm WARN unmet dependency which is version 0.4.6
==> default: npm WARN unmet dependency /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-wrapper/node_modules/download/node_modules/gulp-decompress/node_modules/decompress/node_modules/vinyl-assign requires object-assign@'^3.0.0' but will load
==> default: npm 
==> default: WARN
==> default:  unmet dependency /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle/node_modules/bin-wrapper/node_modules/download/node_modules/object-assign,
==> default: npm WARN
==> default:  unmet dependency which is version 2.1.1
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-jpegtran/node_modules/jpegtran-bin/node_modules/bin-build/node_modules/decompress/node_modules/decompress-tarbz2/node_modules/tar-stream/node_modules/bl requires readable-stream@'~2.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-optipng/node_modules/optipng-bin/node_modules/bin-wrapper/node_modules/download/node_modules/gulp-decompress/node_modules/decompress/node_modules/decompress-tar requires vinyl@'^0.4.3' but will load
==> default: npm WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  WARN unmet dependency /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-optipng/node_modules/optipng-bin/node_modules/bin-wrapper/node_modules/download/node_modules/gulp-decompress/node_modules/decompress/node_modules/decompress-tarbz2 requires vinyl@'^0.4.3' but will load
==> default: npm WARN unmet dependency undefined,
==> default: npm WARN 
==> default: unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-optipng/node_modules/optipng-bin/node_modules/bin-wrapper/node_modules/download/node_modules/gulp-decompress/node_modules/decompress/node_modules/decompress-targz requires vinyl@'^0.4.3' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-optipng/node_modules/optipng-bin/node_modules/bin-wrapper/node_modules/download/node_modules/gulp-decompress/node_modules/decompress/node_modules/decompress-targz/node_modules/tar-stream/node_modules/bl requires readable-stream@'~2.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/imagemin/node_modules/imagemin-svgo/node_modules/svgo/node_modules/js-yaml/node_modules/argparse requires lodash@'>= 3.2.0 < 4.0.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/lodash,
==> default: npm WARN unmet dependency which is version 2.4.2
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/grunt-contrib-imagemin/node_modules/pretty-bytes/node_modules/meow/node_modules/indent-string requires minimist@'^1.1.0' but will load
==> default: npm
==> default:  WARN unmet dependency undefined,
==> default: npm WARN unmet dependency which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/istanbul/node_modules/handlebars/node_modules/uglify-js requires async@'~0.2.6' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency /vagrant/node_modules/istanbul/node_modules/async,
==> default: npm WARN unmet dependency which is version 1.4.2
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/istanbul/node_modules/handlebars/node_modules/uglify-js requires optimist@'~0.3.5' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version undefined
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/karma/node_modules/log4js requires async@'~0.2.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/async,
==> default: npm
==> default:  WARN unmet dependency which is version 0.9.2
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/karma/node_modules/http-proxy/node_modules/utile requires async@'~0.2.9' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/async,
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  which is version 0.9.2
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  /vagrant/node_modules/karma-coverage/node_modules/dateformat/node_modules/meow/node_modules/indent-string requires minimist@'^1.1.0' but will load
==> default: npm
==> default:  
==> default: WARN
==> default:  
==> default: unmet dependency
==> default:  undefined,
==> default: npm
==> default:  
==> default: WARN
==> default:  unmet dependency which is version undefined
==> default: swagger-node-express@2.0.0 node_modules/swagger-node-express
==> default: ├── docco@0.4.0 (commander@2.8.1)
==> default: ├── connect@3.4.0 (utils-merge@1.0.0, parseurl@1.3.0, debug@2.2.0, finalhandler@0.4.0)
==> default: └── lodash@1.3.1
==> default: Installing Bower packages
==> default: sudo
==> default: : 
==> default: bower: command not found
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

If I attempt `vagrant ssh` from that, it results in the following error:

```
sh: 1: grunt: not found

npm ERR! habitrpg@0.0.0-152 start: `grunt run:dev`
npm ERR! Exit status 127
npm ERR! 
npm ERR! Failed at the habitrpg@0.0.0-152 start script.
npm ERR! This is most likely a problem with the habitrpg package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt run:dev
npm ERR! You can get their info via:
npm ERR!     npm owner ls habitrpg
npm ERR! There is likely additional logging output above.
npm ERR! System Linux 3.2.0-23-generic
npm ERR! command "/usr/bin/node" "/usr/bin/npm" "start"
npm ERR! cwd /vagrant
npm ERR! node -v v0.10.37
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /vagrant/npm-debug.log
npm ERR! not ok code 0
```

I've tried installing grunt individually and phantomjs individually, but using `vagrant up` and `vagrant ssh` still results in the error. 

Pie in the sky "I want it to work ASAP!" solution, I wonder...I have an older, 2-month-old clone of the HabitRPG repo in another folder, with a working Vagrantbox. Would it be possible to copy the hidden .vagrant folder from that folder to the newer one and have it install the necessary components that way? I didn't want to try it before getting feedback since I'm sure there have been updates to that folder, but I really just want it to work!

**Comment by abondis** [comment]
after doing vagrant ssh, I ran those commands, I'm pretty sure just `npm install` was necessary but still pasting everything that made it work for me

```
  npm install grunt
  npm install grunt-browserify grunt-contrib-uglify grunt-contrib-cssmin
  npm install request
  npm install gulp-imagemin
  npm install gulp.spritesmith
  npm install gulp-clean
  npm install gulp-nodemon
  npm install amazon-payments
  bower --config.interactive=false install -f
  npm install
```

to figure out the missing dependencies I was running `gulp run:dev` every time, then installing the missing dependency.

**Comment by abondis** [comment]
in the end, http://habitica.wikia.com/wiki/Installation_troubleshooting gives the answer ...

**Comment by mfdeveloper** [comment]
Hello @AnitaYCheng, I had the similar problem with `phatomjs` dependency, and I suggested one fix in issue #5801 . Please, check if this solution work to you :)
